cmd/manager/main.go

```diff
@@ -217,3 +236,66 @@ func serveCRMetrics(cfg *rest.Config, operatorNs string) error {
 	}
 	return nil
 }
+
+const sccName = "neutron-operator"
+
+// EnsureSCCExists ensures the security context constraint for neutron-operator exists
+func ensureSCCExists(c client.Client, saNamespace, saName string) error {
+
+	userName := fmt.Sprintf("system:serviceaccount:%s:%s", saNamespace, saName)
+	trueVar := true
+	priority := int32(10)
+
+	scc := &secv1.SecurityContextConstraints{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: sccName,
+			Labels: map[string]string{
+				"neutron.openstack.org": "",
+			},
+		},
+		Priority: &priority,
+		FSGroup: secv1.FSGroupStrategyOptions{
+			Type: secv1.FSGroupStrategyRunAsAny,
+		},
+		AllowPrivilegedContainer: true,
+		AllowPrivilegeEscalation: &trueVar,
+		AllowHostDirVolumePlugin: true,
+		AllowHostIPC:             true,
+		AllowHostNetwork:         true,
+		AllowHostPID:             true,
+		AllowHostPorts:           true,
```

Comment:
we have this set to 'false' in deploy/scc.yaml.

Reply:
done set to false to have in sync